src/map/lua/lua_baseentity.cpp

```diff
@@ -2808,7 +2808,7 @@ uint16 CLuaBaseEntity::getEquipID(SLOTTYPE slot)
 
 std::optional<CLuaItem> CLuaBaseEntity::getEquippedItem(uint8 slot)
 {
-    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_PC);
+    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype != TYPE_PC && m_PBaseEntity->objtype != TYPE_TRUST);
 
     if (m_PBaseEntity->objtype == TYPE_PC)
```

Comment:
If these whole sections bail if you're not a char, we dont need these debug breaks anymore :D

We can also skip out on wrapping the bodies of these functions with that statement by inverting it and putting it at the top:

```cpp
if (m_PBaseEntity->objtype != TYPE_PC)
{
    return std::nullopt;
}

... the rest
```